compiler/compile/OMRSymbolReferenceTable.cpp

```diff
@@ -713,7 +717,7 @@ OMR::SymbolReferenceTable::findOrCreateCodeGenInlinedHelper(CommonNonhelperSymbo
    {
    if (!element(index))
       {
-      TR::MethodSymbol * sym = TR::MethodSymbol::create(trHeapMemory(),TR_Helper);
+      TR::MethodSymbol * sym = TR::MethodSymbol::create(trHeapMemory(),TR_None);
```

Comment:
Thanks for your answer. So anything in the category of CodeGenInlinedHelper is appropriate for the TR_None linkage convention; there are other examples of helpers that require actual "helper linkage" I assume and that is why TR_Helper is still a category in the new enum correct ?